ktor-features/ktor-i18n/jvm/src/io/ktor/i18n/Translate.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.i18n
+
+import io.ktor.application.*
+import java.util.*
+
+/**
+ * Translate a message key to an accepted language specified in HTTP request
+ */
+
+fun ApplicationCall.translate(key: String): String {
```

Comment:
Looks better and I agree with you, that way is more verbose. But, so your suggestion is to create a map which contains all i18n keys/values?

Reply:
i guess what sergey means is use kotlin's operator overloading functionality instead of a conventional method invocation: `operator fun get(key: String): String`

Reply:
Definitely. Thanks for the suggestion :)

Reply:
Just changed to ``fun i18n(key: String)``. WDYT?